config/environments/production.rb

```diff
@@ -57,11 +58,8 @@
   config.i18n.fallbacks = true
 
   # Enable DNS rebinding protection and other `Host` header attacks.
-  # config.hosts = [
-  #   "example.com",     # Allow requests from example.com
-  #   /.*\.example\.com/ # Allow requests from subdomains like `www.example.com`
-  # ]
-  #
+  config.hosts = HostPatterns.allowed_host_patterns
+
   # Skip DNS rebinding protection for the default health check endpoint.
   # config.host_authorization = { exclude: ->(request) { request.path == "/up" } }
```

Comment:
I think we need to uncomment this as well, so the healthchecks can run locally and message `localhost/up` without errors:

```suggestion
  config.host_authorization = { exclude: ->(request) { request.path == "/up" } }
```